pylua/network.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+import asyncio
+import websockets
+from websockets import connect
+
+async def server_task(websocket, path): #pylint: disable=unused-argument
+    message = await websocket.recv()
+    response = message
+    await websocket.send(response)
+
+def run_websocket_server(address='localhost', port=8087):
+    start_server = websockets.serve(server_task, address, port)
+    asyncio.get_event_loop().run_until_complete(start_server)
+
+
+class EchoWebsocket:
+    def __init__(self, address):
+        self._address = address
+
+    async def __aenter__(self):
+        self._conn = connect(self._address) #pylint: disable=attribute-defined-outside-init
```

Comment:
@OHerasym, try to avoid suppression in general. In this case you just have to set to null conn and websocket in init. please, do this.